apps/spikeConverter.cpp

```diff
@@ -62,16 +62,16 @@ int main( int argc, char* argv[] )
 #endif
         ( "output,o", po::value< std::string >()->default_value( "out.spikes" ),
           uriHelp.c_str( ));
-    
+
     po::positional_options_description positional;
     positional.add("input", 1);
-    positional.add("output", 2);    
-    
+    positional.add("output", 2);
+
     po::variables_map vm;
     try
     {
         po::store( po::command_line_parser( argc, argv ).
-                   options( options ).positional( positional ).run(), vm );       
```

Comment:
I did the edition inside Github itself. Could I have pushed on top of your branch from the command line?